content/docs/tinacms-context.md

```diff
@@ -84,36 +78,39 @@ import { TinaAdmin } from 'tinacms';
 export default TinaAdmin;
 ```
 
-### Manually toggling via `useEditState`
+## Making content contextually editable with `useTina`
 
-You can manually enter and exit edit mode by tapping into the `useEditState` hook. A common pattern is to place this hook on an "admin" page, which simply puts you into edit mode and sends you back to the page you were on.
+Contextual editing can be setup on a page with the `useTina` hook
 
-```tsx
-// pages/admin.js
-import { useEffect } from 'react'
-import { useRouter } from 'next/router'
-
-import { useEditState } from 'tinacms/dist/edit-state'
-
-const GoToEditPage = () => {
-  const { editState, setEdit } = useEditState()
-  const router = useRouter()
-  useEffect(() => {
-    setEdit(!editState)
-    // Go back to the page you were on previously
-    router.back()
-  }, [])
-  // Display a brief message to the user
-  return <div>Going into edit mode...</div>
-}
+```jsx
+// ...
+import { useTina } from 'tinacms/dist/edit-state'
+
+const query = `{
+  getPageDocument(relativePath: "home.mdx"){
+    data{
+      body
+    }
+  }
+}`
 
-export default GoToEditPage
+export default function Home(props) {
```

Comment:
I feel like this page should show the full example (including `getStaticProps` which uses `statisRequest`) so readers can see how the two work together. Without it, it seems like `useTina` does the data fetching.